static/app/components/smartSearchBar/index.tsx

```diff
@@ -74,8 +72,6 @@ const ACTION_OVERFLOW_STEPS = 75;
 
 const makeQueryState = (query: string) => ({
   query,
-  // Anytime the query changes and it is not "" the dropdown should show
-  showDropdown: true,
```

Comment:
One change in behavior I found because of this is that if you esc out to close the dropdown and start typing again the dropdown doesn't show again unlike before when it does show again when you start typing. Do we want to add this functionality back in but instead on key down/up?

Reply:
Ah good catch! Yeah that's a behavior we want to retain. I'll add something in to fix this